<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoodDetail</class>
 <widget class="QDialog" name="FoodDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/file/images/back8.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>346</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>266</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>菜名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="styleSheet">
         <string notr="true">color:white;
background:transparent;
border:none;
border-bottom:1px solid #FFFFFF;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>价格：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_price">
        <property name="styleSheet">
         <string notr="true">color:white;
background:transparent;</string>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="value">
         <double>66.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>特价率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_rate">
        <property name="styleSheet">
         <string notr="true">color:white;
background:transparent;</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>0.990000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.800000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>是否特价：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>分类：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="styleSheet">
         <string notr="true">color:white;
background:transparent;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_ok">
        <property name="text">
         <string>确认修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
